eng/sdl-tsa-vars.config

```diff
@@ -0,0 +1,9 @@
+-SourceToolsList @("policheck","credscan")
+-TsaInstanceURL https://devdiv.visualstudio.com/
+-TsaProjectName DEVDIV
+-TsaNotificationEmail aspnetcore-build@microsoft.com
+-TsaCodebaseAdmin REDMOND\kevinpi
+-TsaBugAreaPath DevDiv\ASP.NET Core
+-TsaIterationPath DevDiv
+-TsaOnboard $True
+-TsaPublish $True
```

Comment:
Add `-TsaRepositoryName "AspNetCore-Tooling"	-TsaCodebaseName "AspNetCore-Tooling"`

Reply:
I thought you said here that we don't need these? https://github.com/dotnet/aspnetcore/pull/21153#discussion_r414036999

Reply:
Ah, never mind, I see what you're saying. Will fix this in my other PRs too